### PR TITLE
chore: update footer on webbsite

### LINKED ISSUE #1128: [CHECKLIST] Update footer component for Webbsite
## Description

Please see below for the latest updates to the footer component for Webbsite

- [ ] Update footer component to include navigation links and site map on Webbsite
- [ ] Add Linkedin & YouTube social icons to footer component
- [ ] Remove the requirement for users to enter their name for subscribing to email list
- [ ] Update mobile footer component with above changes

Image

Figma Access:
https://www.figma.com/file/d5T0P9k4anUGdBlqgh4ueW/%F0%9F%96%A5%EF%B8%8F-Webb-Website?node-id=1028%3A32475&t=efsptjtm5FlBowH5-1

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes
Update Footer according to the task's requirement

### Proposed area of change

- [ ] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [x] `apps/webbsite`
- [ ] `apps/faucet`
- [ ] `apps/tangle-website`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
_Specify any issues that can be closed from these changes (e.g. Closes #233)._
- Closes #1128 

### Screen Recording
_If possible provide a screen recording of proposed change._

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx [hunk 1]
```diff
@@ -49,6 +150,11 @@ const socials: Array<SocialType> = [
     Icon: GithubFill,
     href: 'https://github.com/webb-tools',
   },
+  {
+    name: 'YouTube',
+    Icon: YouTubeFill,
+    href: 'https://github.com/webb-tools',
```
Comment (devpavan04) [inline libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx:156]: I think we can point this to blog videos for now. What do you think @dutterbutter?

https://webb.tools/blog/videos

**Comment by dutterbutter** [comment]
> @dutterbutter can you help me with this error on the privacy-policy and terms-and-conditions pages of the Webbsite when it runs the build command. Thank you!
> 
> Screenshot 2023-05-01 at 20 20 55

I will investigate!

File: libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx [hunk 2]
```diff
@@ -13,16 +15,110 @@ import {
...
+  >;
+};
 
 type SocialType = {
   Icon: (props: IconBase) => JSX.Element;
   name: string;
   href: string;
 };
 
+const navLinks: Array<NavLinkType> = [
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx:34]: Can we make the following changes:

- [ ] Replace "Resources" with "Community" and include the same links as the docs site
- [ ] Can we replace "Network" with "Ecosystem", include the links as the docs site, and remove "DAPP" column
- [ ] Add "Legal" column and include the privacy and terms links

File: libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx [hunk 3]
```diff
@@ -13,16 +15,125 @@ import {
...
+        isInternal: false,
+      },
+    ],
+  },
+  {
+    group: 'Ecosystem',
+    links: [
+      {
+        label: 'Tangle',
+        url: 'https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftangle-standalone1.webb.tools%2F#/explorer',
```
Comment (dutterbutter) [inline libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx:70]:
```suggestion
        url: 'https://polkadot.js.org/apps/?rpc=wss://tangle-standalone-archive.webb.tools#/explorer',
```

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**